udata/models/datetime_fields.py

```diff
@@ -63,5 +63,6 @@ class Datetimed(object):
 
 @pre_save.connect
 def set_modified_datetime(sender, document):
-    if isinstance(document, Datetimed):
+    from udata.models import Dataset
```

Comment:
Not proud of this, maybe not inheriting from `Datetimed` is more clever after all.